db/sql/src/logs.rs

```diff
@@ -346,6 +346,58 @@ impl HoprDbLogOperations for HoprDb {
             })
             .await
     }
+
+    async fn ensure_logs_origin(&self, contract_address_topics: Vec<(Address, Hash)>) -> Result<()> {
```

Comment:
Is this information not known at DB creation time?

Reply:
No.

Reply:
But this is still an application call to the database that is a pre-requisite to its usage. And the topics are known when the application starts, so could be easily delivered to the DB then, is that correct?

Reply:
The topics are not known when the application starts as it does not have any clue about "topics" at that point.